ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-postsubmits.yaml

```diff
@@ -10,8 +10,8 @@ postsubmits:
       containers:
       - args:
         - --artifact-dir=$(ARTIFACTS)
-        - --secret-dir=/usr/local/e2e-azure-secret
-        - --template=/usr/local/e2e-azure
+        - --target=[images]
```

Comment:
on pre-submit we have only ` --target=[output:stable:sync]` do we need images there too?

Reply:
Yeah, we likely need that, let me add it

Reply:
After testing locally, it doesn't seem it's required so leaving it out.

Reply:
lets test in prod :D